src/elm/Main.elm

```diff
@@ -673,6 +699,34 @@ subs model =
 --
 
 
+{-| Check if we have period tickets
+@TODO - Reusable as utils?
+-}
+hasPeriodTickets : Shared -> Bool
+hasPeriodTickets shared =
+    shared.fareProducts
+        |> List.any
+            (\product ->
+                product.type_
+                    == ProductTypePeriod
+                    && List.member DistributionChannelWeb product.distributionChannel
+            )
+
+
+{-| Check if we have carnet tickets
+@TODO - Reusable as utils?
```

Comment:
Usikker på hvor best å gjøre av slike ting. Det er strengt tatt ikke Util. Sliter med slike domenekoblinger i en lagbasert mappestruktur

Reply:
I `Shared`-modulen kanskje? Så at dette var duplisert i `Overview`, så det ville i hvert fall vært mer naturlig å legge det nærmere datastrukturen når det ikke er noe som er unikt for kun 1 modul.